src/fromager/dependencies.py

```diff
@@ -24,10 +24,23 @@ def get_build_system_dependencies(
     logger.info(
         f"{req.name}: getting build system dependencies for {req} in {sdist_root_dir}"
     )
+
+    build_system_req_file = sdist_root_dir.parent / "build-system-requirements.txt"
+    if build_system_req_file.exists():
+        logger.info(
+            f"{req.name}: {build_system_req_file} already exists. Loading build system dependencies from {build_system_req_file.name}"
```

Comment:
This log message will include the filename twice, right?

```suggestion
            f"{req.name}: loading build system dependencies from {build_system_req_file}"
```